### PR TITLE
feat: add the Key Ceremony component

### PR DESCRIPTION
**Pull request by beagleknight**
This adds a new class called `KeyCeremonyComponent` that will help integrating the key ceremony process in any UI. I used this class in the sandbox environment and also tested in `decidim`.

The idea is to delete some code that was duplicated between the two environments and make it easier to understand.

### CODE REVIEW

File: decidim-bulletin_board-js/src/key-ceremony/key-ceremony.component.js [hunk 1]
```diff
@@ -0,0 +1,118 @@
...
+ * This class is used to bind any UI elements to a key ceremony process.
+ */
+export class KeyCeremonyComponent {
+  /**
+   * Initialises the class with the given params.
+   * @param {Object} params - An object that contains the initialization params.
+   *  - {Object} bulletinBoardClientParams - An object to configure the bulletin board client.
+   *  - {String} electionUniqueId - The unique identifier of an election.
+   *  - {String} trusteeUniqueId - The unique identifier of a trustee.
+   *  - {Object} trusteeIdentificationKeys - A object that contains both the public and private key for
```
Comment (slickepinne) [inline decidim-bulletin_board-js/src/key-ceremony/key-ceremony.component.js:15]:
```suggestion
   *  - {Object} trusteeIdentificationKeys - An object that contains both the public and private key for
```

File: decidim-bulletin_board-js/src/key-ceremony/key-ceremony.component.js [hunk 2]
```diff
@@ -0,0 +1,118 @@
...
+      bulletinBoardClient,
+      identificationKeys: trusteeIdentificationKeys,
+      election: this.election,
+    });
+  }
+
+  /**
+   * Bind UI events to the key ceremony process.
+   *
+   * @param {Object} eventCallbacks - An object that contains event callbacks functions.
```
Comment (slickepinne) [inline decidim-bulletin_board-js/src/key-ceremony/key-ceremony.component.js:43]:
```suggestion
   * @param {Object} eventCallbacks - An object that contains event callback functions.
```
😂

**Review by orlera — APPROVED** [review]
⬛

**Review by slickepinne — APPROVED** [review]
That looks so clean! Amazing! 🎉

**Review by leio10 — APPROVED** [review]
Amazing! 👏 💎

**Pull request merged**